.github/workflows/python.yml

```diff
@@ -39,52 +47,39 @@ permissions:
     contents: read
 
 jobs:
-    load-engine-matrix:
+    get-matrices:
         runs-on: ubuntu-latest
+        # Avoid running on schedule for forks
+        if: (github.repository_owner == 'valkey-io' || github.event_name != 'schedule')
         outputs:
-          matrix: ${{ steps.load-engine-matrix.outputs.matrix }}
+            engine-matrix-output: ${{ steps.get-matrices.outputs.engine-matrix-output }}
+            host-matrix-output: ${{ steps.get-matrices.outputs.host-matrix-output }}
+            version-matrix-output: ${{ steps.get-matrices.outputs.version-matrix-output}}
         steps:
-            - name: Checkout
-              uses: actions/checkout@v4
-        
-            - name: Load the engine matrix
-              id: load-engine-matrix
-              shell: bash
-              run: echo "matrix=$(jq -c . < .github/json_matrices/engine-matrix.json)" >> $GITHUB_OUTPUT
-        
-    test:
+            - uses: actions/checkout@v4
+            - id: get-matrices
+              uses: ./.github/workflows/create-test-matrices
+              with:
+                  language-name: python
+                  dispatch-run-full-matrix: ${{ github.event.inputs.full-matrix || 'false' }}
+
+    test-python:
         runs-on: ${{ matrix.host.RUNNER }}
-        needs: load-engine-matrix
+        needs: get-matrices
         timeout-minutes: 35
         strategy:
             fail-fast: false
             matrix:
-                engine: ${{ fromJson(needs.load-engine-matrix.outputs.matrix) }}
-                python:
-                  # - "3.8"
-                  # - "3.9"
-                  # - "3.10"
-                  # - "3.11"
-                    - "3.12"
-                host:
-                    - {
-                        OS: ubuntu,
-                        RUNNER: ubuntu-latest,
-                        TARGET: x86_64-unknown-linux-gnu
-                    }
-                    # - {
-                    #     OS: macos,
-                    #     RUNNER: macos-latest,
-                    #     TARGET: aarch64-apple-darwin
-                    # }
-
+                engine: ${{ fromJson(needs.get-matrices.outputs.engine-matrix-output) }}
+                python: ${{ fromJson(needs.get-matrices.outputs.version-matrix-output) }}
+                host: ${{ fromJson(needs.get-matrices.outputs.host-matrix-output).include }}
```

Comment:
same - what .include is?

Reply:
why we have it only in some of the `needs.get-matrices.outputs.host-matrix-output`?